### DISCUSSION TITLE
Plugins Stats

### DISCUSSION THREAD

**Discussion by ckdarby** [godotengine/godot-proposals#8759] [General Discussions]
It would be nice if the editor returned some anonymous stats about plugins to the Godot Foundation.

Some of the ideas I had, but not limited to:
- Most Installed
- Trending
- Tracking plugin version install base
- Tracking plugins running on Godot versions

Here is a breakdown of why we should care about each of these.

**Most Installed**
It makes it easier to find common plugins, which is also a proxy for well-established and almost "core engine accepted" plugins.

**Trending**
Finding up-and-coming plugins to test out and provide feedback is easier.

**Tracking plugin version install base**
It helps plugin developers know what versions their user base is using in terms of their plugin version.

**Tracking plugins running on Godot versions**
It helps plugin developers know what to continue supporting moving forward and to make data-driven decisions around cutting off Engine support and knowing the impact.

**Comment by ckdarby** [comment]
Technically, a much larger conversation for the foundation would be to become a Godot registry.

This would open up the possibility of a "marketplace" funnelled through the foundation, which could take % of transactions and generate cash flow for the foundation to succeed forever.

**Comment by AThousandShips** [comment]
See also:
* https://github.com/godotengine/godot-proposals/issues/8114

For tracking downloads I'm not sure what various data tracking policies would apply and how it'd affect how we could do this

Tracking what plugins are running on different users editors goes against the general design and decisions concerning that at least currently, the editor does not communicate with anything and that's by design (and I for one would vehemently oppose any tracking being added to the editor)
